app/assets/javascripts/questions.js

```diff
@@ -10,4 +10,16 @@ $(document).ready(function(q){
       $('.other-question').removeClass('hidden');
     });
 
-});
\ No newline at end of file
+// Runs when the page loads and disables the submit button. Checks that both the answer and question text boxes are blank
+  if ($('#question_text').val() || $('#question_answers_attributes_0_text').val() == "") 
+   {$('#submit_question').attr('disabled', true);}
+
+// Every time a key is pressed, this code runs and checks the contents of the answer and question text boxes. If both are full then the submit button is enabled 
+  $('#question_text, #question_answers_attributes_0_text').keyup(function(){
+    if($('#question_text').val() && $('#question_answers_attributes_0_text').val() !=  "") 
+         $('#submit_question').attr('disabled', false);    
+    else
+         $('#submit_question').attr('disabled', true);   
+  });
```

Comment:
These two blocks can also be refactored exactly like the ones above, in `answers.js`, because they're doing the same check. (I'll leave this one for you to figure out, but it's essentially exactly like the answers.js example I gave.)

Since you want it to work whether it's being called for the first time or any subsequent time, use the code that includes the `else` clause in your function.

Also:
1. Did you test this after you changed the code? To me it looks like there's an error in the `if` statement.
2. Please add brackets for your `if-else` block:
```
if ( condition ) {
  ... do something ...
} else {
  ... do a different thing ...
}
```